docs/3_finetuner_in_action/3_3_text_to_image.md

```diff
@@ -49,14 +51,20 @@ run = finetuner.fit(
         multi_modal=True,
     )
 ```
-Let's understand what this piece of code does.
-We start with providing `model`, `run_name`, names of training and evaluation data. We also provide some hyper-parameters
-such as number of `epochs` and a `learning_rate`. Additionally, we use `BestModelCheckpoint` to save the best model after each epoch and `EvaluationCallback` for evaluation. Now let's move on to CLIP-specific arguments. We provided `image_modality`
-and `text_modality` which are needed for `CLIP` model to distribute data across its two models properly. (More on this in the [create training data](../2_step_by_step/2_4_create_training_data.md) section).
+Let's understand what this piece of code does:
+```{admonition} finetuner.fit parameters
+:class: tip
+The only required arguments are `model` and `train_data`. We provide default values for others. Here is the [full list of the parameters](../../api/finetuner/#finetuner.fit). 
+```
+* We start with providing `model`, `run_name`, names of training and evaluation data.
+* We also provide some hyper-parameters such as number of `epochs` and a `learning_rate`.
+* Additionally, we use `BestModelCheckpoint` to save the best model after each epoch and `EvaluationCallback` for evaluation.
```

Comment:
add some developer references here for the backticks (same as other guide comments)